src/titiler/xarray/tests/test_factory.py

```diff
@@ -165,21 +165,21 @@ def test_tiles(filename, app):
 
     resp = app.get(
         "/md/tiles/WebMercatorQuad/0/0/0",
-        params={"url": filename, "variable": "dataset", "rescale": "0,500"},
+        params={"url": filename, "variable": "dataset", "rescale": "0,500", "bidx": 1},
```

Comment:
why do you need to add `bidx=1` now? Is it because the reader will return a 3D array now with the new `sel` interface (instead of 2D with `drop_dim`)?

Reply:
Same question as I asked, basically: https://github.com/developmentseed/titiler/pull/1134#discussion_r2055602147

Reply:
> why do you need to add bidx=1 now? Is it because the reader will return a 3D array now with the new sel interface (instead of 2D with drop_dim)?

Exactly, also before we were always selecting the first time value. Now we can have 3D arrays